### PR TITLE
Fixes for #181 : adding introduction and some concept

### PR DESCRIPTION
**Pull request by gauravzqa**
added some basic introduction and concepts explanation for snare.
fixes for https://github.com/mushorg/snare/issues/181

### CODE REVIEW

File: docs/quick-start.rst [hunk 1]
```diff
@@ -3,12 +3,25 @@ SNARE
 
 Super Next generation Advanced Reactive honEypot
 
+Snare is a web application honeypot and is the successor of Glastopf, which has many of the same
+features as Glastopf as well as ability to convert existing Web pages into attack surfaces with TANNER.
```
Comment (rnehra01) [inline docs/quick-start.rst]: Create a hyperlink for TANNER pointing to tanner repo.

File: docs/quick-start.rst [hunk 2]
```diff
@@ -3,12 +3,25 @@ SNARE
...
+SNARE is an abbreviation for Super Next generation Advanced Reactive HonEypot. Every event sent from
+SNARE to TANNER is evaluated, and TANNER decides how SNARE should respond to the client. This allows
```
Comment (rnehra01) [inline docs/quick-start.rst]: Here too.

File: docs/quick-start.rst [hunk 3]
```diff
@@ -3,12 +3,25 @@ SNARE
...
+
 Basic Concepts
 """"""""""""""
 
 * Surface first. Focus on the attack surface generation.
 * Sensors and masters. Lightweight collectors (SNARE) and central decision maker (tanner).
 
+SNARE provides a basic Web interface that displays the number of attack sessions, total duration,
+and the frequencies of types of attacks. SNARE also provides further information such as IPs,
+ports, user agents, start and end times, paths and attack types on each individual session.
```
Comment (rnehra01) [inline docs/quick-start.rst]: I don't think Snare does that. This information is shown by TannerWeb.

**Comment by Aniket21mathur** [comment]
@rnehra01 I have made the required changes in #223. Please review.

**Comment by afeena** [comment]
#223

**Pull request closed without merging**